src/Google.Cloud.Metadata.V1/AccessConfigurationMetadata.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Net;
+
+namespace Google.Cloud.Metadata.V1
+{
+    /// <summary>
+    /// Metadata for a network interface's access configurations.
+    /// </summary>
+    /// <seealso cref="NetworkInterfaceMetadata.AccessConfigurations"/>
+    public sealed class AccessConfigurationMetadata
+    {
+        /// <summary>
+        /// Gets the public IP address to use for external traffic or <seealso cref="IPAddress.None"/> if the network interface
```

Comment:
`seealso` is usually used as a standalone element after the summary. You want `<see>` here.